doc/amiga/aga/AGA.guide

```diff
@@ -5,7 +5,7 @@
 ## $VER: AGA.guide 1.0 (29.Aug.93)
 ## Done by T.F.A. 1993
 ##
-## Pandore ChipSet AmigaGuide� documentation.
+## Pandore ChipSet AmigaGuide® documentation.
```

Comment:
don't know why this line came in. On my mac it looks like the diff "fixes" a broken (R) char, but feel free to not include that edit.